libraries/standard/http/cbmc/proofs/findHeaderFieldParserCallback/Makefile

```diff
@@ -0,0 +1,20 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY=harness
+HARNESS_FILE=findHeaderFieldParserCallback_harness
+
+MAX_HEADER_FIELD_LENGTH=10
```

Comment:
Comment about why we bound the header field? I leave these for new people coming into CBMC.  
https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1079#discussion_r462531641